kata-deploy/kata-deploy.yaml

```diff
@@ -41,6 +41,8 @@ spec:
           mountPath: /var/run/dbus
         - name: systemd
           mountPath: /run/systemd
+        - name: local-bin
+          mountPath: /usr/local/bin/
```

Comment:
I may be misunderstanding what this does, but what if you already have files in `/usr/local/bin/`? We don't "own" this directory so maybe we should be creating `/usr/local/kata/bin/` and linking files into `/usr/local/bin/`?

Reply:
This allows the container workload to be able to write to the path. In our case, the container will create a symlink back to /opt/kata/bin, and if there was the particular binary already there (not expected, this is a kata runtime binary), it’ll move it to “backup”.  The file has to be within path of the host system. 


This is a temporary issue for containerd. 

This clarify?

Reply:
Thanks.

Reply:
@egernst what happens if this directory does not exist (as the case is with clearlinux)? Have you tested that condition.

Reply:
I *think* it'll create it, but rather than work on thoughts and feelings, let me give it a quick test, @mcastelino !

Reply:
@mcastelino it will create the path on the host.

Reply:
The path does include /usr/local/bin in CL. So you are good